site/src/quacs-rs/Cargo.toml

```diff
@@ -25,7 +25,7 @@ console_error_panic_hook = { version = "0.1.6", optional = true }
 wasm-bindgen-test = "0.3"
 
 [package.metadata.wasm-pack.profile.release]
-wasm-opt = ["-O3"]
+wasm-opt = false
```

Comment:
I understand you need this for your local builds, but this should not be committed to the repo

Reply:
Yes you are right sorry about that. I did change this but it is on my forked repository. Do I have to make another pull request or is there an easy way for this to be edited? Sorry I am not thoroughly familiar with GitHub.

Reply:
Looks like you fixed it on your next commit. Looks good now.

Reply:
Ignore my previous comment if the changes have already been made to the file

Reply:
Ok sounds good, thanks!